assets/scss/texts.scss

```diff
@@ -91,6 +91,7 @@ header .col {
   /deep/ .authored-section .col-lg-9 {
     flex: 0 0 100%;
     max-width: 100%;
+    padding: 0;
```

Comment:
This doesn't seem to actually resolve the styling issue on the text.
They're still center aligned but with empty space on the right side.
Do we need to move these styles back into the component/pages for the /deep/ selector to function properly?

Reply:
resolved by removing /deep/ selectors